products/openzfs.md

```diff
@@ -0,0 +1,53 @@
+---
+permalink: /openzfs
```

Comment:
```suggestion
permalink: /openzfs
alternate_urls:
  - /zfs
```

Reply:
This could open a can of worms similar to Java. There's multiple different ZFS projects, like Oracle ZFS for example.

Reply:
It does a temporary redirect, so we can always re-arrange these. Since there's only one zfs page for now, redirecting here makes sense. Later, we can pick the most popular one?

Reply:
Sounds good.